src/components/MessageList/MessageList.jsx

```diff
@@ -62,11 +63,12 @@ const MessageList = ({
       const isOnline =
         listOfOnlineUsers.get(message.user.id.toString()) === true;
 
-      const isLastReadMessage = index === sortedMessages.length - 1;
+      const isLastVisibleReadMessage =
+        index === sortedMessages.length - unreadMessages.length - 1;
       return (
         <div
           key={message.id || message.creationDate}
-          ref={isLastReadMessage ? lastReadMessageRef : null}
+          ref={isLastVisibleReadMessage ? lastVisibleReadMessageRef : null}
```

Comment:
Can it works here `ref={isLastVisibleReadMessage && lastVisibleReadMessageRef}`?

Reply:
If I use 'ref={isLastVisibleReadMessage && lastVisibleReadMessageRef}' I have error and it doesn't work correct,  because it passes false to the ref when the condition is false. But React expects ref = object or null, not boolean

Reply:
Ok, I see